views/stanford/footer.pug

```diff
@@ -24,9 +22,17 @@ div.container#stanford-footer
         div.col-md-2
           a(href='https://github.com/stanford-oval/almond-cloud/issues')= _("Report a bug")
         div.col-md-2
+          a(href='https://exploredegrees.stanford.edu/nonacademicregulations/nondiscrimination/')= _("Non-Discrimination")
+        div.col-md-2.footer-icons
           a(href='https://twitter.com/StanfordOVAL').social-icon
             img(src=Config.ASSET_CDN + '/images/symbolic-logo-twitter.svg', title=_("Connect with us on Twitter"))
           span.social-icon-spacer(role='presentation')
+          a(href='https://discord.gg/anthtR4').social-icon
+            img(src=Config.ASSET_CDN + '/images/discord_w.svg', title=_("Join us on Discord"))
```

Comment:
Can we keep the existing naming convention for these icons? Makes it easier to see what each file is used for.

Reply:
no problem, didn't noticed a "convention" , so thought that the name was clear enough, will fix